services/dy-2Dgraph/use-cases/cc/cc-oned/cc_oned.ipynb

```diff
@@ -195,7 +208,7 @@
     "    num_cells = 165\n",
```

Comment:
I wonder fi some validation could be done on the data with human readable error messages ... so we do not just get a nasty python error